spec/features/user/show.html.erb_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe 'User show page', type: :feature do
+  describe 'show page' do
+    before :each do
+      # @user = User.create(name: 'Tom', bio: 'Teacher from Mexico.', photo: 'https://unsplash.com/photos/F_-0BxGuVvo')
+      # @id = @user.id
+      # Post.create(user_id: @user, title: 'Hello', text: 'Make a new comment on this post')
+      # Post.create(user_id: @user, title: 'hello', text: 'Make a new comment on this post')
+      visit '/users/83'
+    end
+
+    it 'show username' do
+      expect(page).to have_content('Tom')
+    end
+
+    it 'show page title' do
+      expect(page).to have_content('User information')
+    end
+
+    it 'show the user bio' do
+      expect(page).to have_content('Teacher from Mexico')
+    end
+
+    it 'show the number of post' do
+      expect(page.body).to include('Number of post: 1')
+    end
```

Comment:
- [ ] This should test that you can see the user's first 3 posts.

Reply:
- [ ] @team as previously requested kindly change this test to show the user's first 3 posts, this is a project requirement
![image](https://user-images.githubusercontent.com/61408860/203868833-8edfcf57-168d-4a92-aefa-1d1864c6fd80.png)